redwood-dom-testing/src/commonMain/kotlin/app/cash/redwood/dom/testing/DomSnapshotter.kt

```diff
@@ -37,16 +38,24 @@ public class DomSnapshotter @PublishedApi internal constructor(
     element: Element,
     name: String = "snapshot",
     scrolling: Boolean = false,
+    width: Int? = null,
+    height: Int? = null,
   ) {
+    element.setAttribute(
+      "style",
+      "${width?.let { "width: ${it}px;" } ?: "width:max-content;"}" +
+        "${height?.let { "height: ${it}px;" } ?: "height:max-content;"}",
```

Comment:
```suggestion
      "width: ${width?.let { "${it}px" } ?: "max-content"}; " +
        "height: ${height?.let { "${it}px" } ?: "max-content"};",
```
Just a style tweak to reduce a little duplication